### ISSUE TITLE
Can not play videos on Fedora.

### ISSUE THREAD

**Issue by Dazawassa** [jeffshee/gnome-ext-hanabi#120]
**Description**
When the plugin is enabled on Fedora 39 it makes the background black and will not play the video, Even when  GTK is installed.

**OS and Shell Information**
Please provide your OS name and version, GNOME Shell version, and the session type (X11/Wayland).
 - OS: [Fedora 39]
 - GNOME Shell: [45.5]
 - Session type: [Wayland]

**Steps to Reproduce**
Please provide the steps to reproduce the bug you are encountering.
1. Install Hanabi.
2. Enable extension and select a video.

**Expected Behavior**
Video live wallpaper plays.

**Actual Behavior**
Background is black no matter what settings or dependencies.

**Additional Information**

**Comment by BALVEY7** [comment]
Check this issue where there is a solution to your problem which in fact is not a bug but you missing some encoding
https://github.com/jeffshee/gnome-ext-hanabi/issues/119
Last comment kinda explains and gives you idea what you need to do

This is for arch not Fedora so update command could be different same as installation

**Comment by jeffshee** [comment]
@Dazawassa Hi, you can refer to this webpage for installing multimedia codecs on Fedora:
https://rpmfusion.org/Howto/Multimedia

tl;dr
```
# Enable RPM Fusion repo
sudo dnf install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://mirrors.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm

# Enable openh264 repo
sudo dnf config-manager --enable fedora-cisco-openh264

# Switch to full ffmpeg
sudo dnf swap ffmpeg-free ffmpeg --allowerasing

# Install multimedia codecs
sudo dnf groupupdate multimedia
```

**Comment by Dazawassa** [comment]
Thanks it solved the issue.
